Differential Expression Analysis Module
========================================

.. autofunction:: lotus.workflows.deg_analysis.marker_genes

   **Citation:** If you use cplearn-based marker gene identification, please cite the `cplearn <https://github.com/csmukherjee/cplearn>`_ package and the CoreSPECT paper (see :doc:`../citations`).

.. autofunction:: lotus.workflows.deg_analysis.rank_genes_groups

   **Citation:** F. Wilcoxon. *Individual comparisons by ranking methods.* Biometrics Bulletin 1, 80-83 (1945).

.. autofunction:: lotus.workflows.deg_analysis.filter_rank_genes_groups

   **Citation:** Based on `scanpy <https://scanpy.readthedocs.io/>`_. F. Alexander Wolf, Philipp Angerer, and Fabian J. Theis. *SCANPY: large-scale single-cell gene expression data analysis.* Genome Biology 19, 15 (2018). `doi:10.1186/s13059-017-1382-0 <https://doi.org/10.1186/s13059-017-1382-0>`_

.. autofunction:: lotus.workflows.deg_analysis.marker_gene_overlap

   **Citation:** Based on `scanpy <https://scanpy.readthedocs.io/>`_. F. Alexander Wolf, Philipp Angerer, and Fabian J. Theis. *SCANPY: large-scale single-cell gene expression data analysis.* Genome Biology 19, 15 (2018). `doi:10.1186/s13059-017-1382-0 <https://doi.org/10.1186/s13059-017-1382-0>`_

.. autofunction:: lotus.workflows.deg_analysis.pick_groups
